utils/encrypt_decrypt.ts

```diff
@@ -1,55 +1,44 @@
-import * as crypto from 'crypto';
+import * as jose from 'node-jose';
 
-export const generateSecretKey = (): string => {
-    return crypto.randomBytes(32).toString('hex'); // 256-bit key
+interface KeyPair { 
+    publicKey: string;
+    privateKey: string;
+}
+const generateJWKKeyPair = async (): Promise<KeyPair> => {
```

Comment:
Add a comment here stating that this will not be used in the code anywhere. It is only added here so that we can manually run it later to generate new public-private key pairs.